### PR TITLE
Add axios example

### PR DESCRIPTION
**Pull request by cherniavskii**
Looking forward to use SWR in one of my projects!

### CODE REVIEW

File: examples/basic-axios/pages/api/data.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+
+const projects = [
+  'facebook/flipper', 'vuejs/vuepress', 'rust-lang/rust', 'zeit/next.js'
+]
+
+export default (req, res) => {
```
Comment (clentfort) [inline examples/basic-axios/pages/api/data.js]: From my understanding of the code the fetcher-function must return an awaitable. This returns `undefined` and will not work.

Comment (cherniavskii) [reply examples/basic-axios/pages/api/data.js]: > fetcher-function must return an awaitable

Not really.
Note, that this is nodejs server handler, and it doesn't matter what is returned - the key part is `res.json` call, which ends the request

Comment (clentfort) [reply examples/basic-axios/pages/api/data.js]: My bad, I looked at the wrong file. Sorry.

Comment (cherniavskii) [reply examples/basic-axios/pages/api/data.js]: No worries :)

File: examples/basic-axios/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link'
+import axios from '../libs/axios'
+import useSWR from '@zeit/swr'
+
+export default () => {
+  const { data } = useSWR('/api/data', axios)
```
Comment (hg-pyun) [inline examples/basic-axios/pages/index.js]: How about add error handling example?

**Pull request closed without merging**